components/03-sections/01-accordion/accordion.scss

```diff
@@ -86,7 +87,22 @@
 }
 
 .accordion .accordion-body-content {
+	padding-left: 85px;
 	box-sizing: content-box;
+	max-width: 85%;
+	width: 100%;
+}
+
+.accordion .accordion-body-content h5 {
+	color: $orange-a11y;
+	font-size: 28px;
+	font-weight: 400;
+	font-family: "open sans", sans-serif;
+}
+
+.accordion-body-content p {
+	margin-bottom: 30px;
+	margin-top: 30px;
 }
 
```

Comment:
yup, I had removed them, the 85% width and additional padding doesn't seem to make sense here.  we can keep the color and font for the heading tho.  What happens when they use more than an h5, like and h2, h3, or h4?

Reply:
Made a new commit based on your feedback. Please review and let me know if I got everything right. Or you can edit it yourself... Just let me know this time, please so I don't try to fix your edits. :)